### PR TITLE
Fix sending tx/sale participation

### PR DESCRIPTION
**Pull request by evgenyboxer**
**What current issue(s) from Trello/Github does this address?**
#1284

**What problem does this PR solve?**
This PR fixes the `getBalance` selector to get the balances from the correct object path. This means that sending transactions did not work too.

`balanceActions` is now renamed to `dashboardActions` as it is only used in Dashboard component and has more than one action. I also removed `blockHeightActions` as it is not used anymore.

### CODE REVIEW

**Comment by evgenyboxer** [comment]
@drptbl

File: app/actions/balancesAndBlockHeightActions.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ type Props = {
   tokens: Array<TokenItemType>
 }
 
-export const ID = 'BALANCES'
+export const ID = 'BALANCES_AND_BLOCKHEIGHT'
```
Comment (comountainclimber) [inline app/actions/balancesAndBlockHeightActions.js]: 👍

**Comment by comountainclimber** [comment]
Nice job fixing this @evgenyboxer looks good to me... Im going to take it on myself to increase coverage in testing specifically related to the block height stuff.

**Comment by drptbl** [comment]
@evgenyboxer Tested, looks good. Thanks!

**Pull request merged**